tests/validate_samples

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+echo "Validating sample data"
+for file in samples/**/TIDES/datapackage.json; do
+    echo "Validating: $file"
+    frictionless validate --schema-sync $file
```

Comment:
I think `**` should be `*`, as is the validation fails (with no samples folder) and the output of echo is that `samples/**/TIDES/datapackage.json` is considered a file.

Reply:
Also, the glob by default returns something if there's no match, need to set nullglob to not run if there's no sample data, e.g.,
```
#!/usr/bin/env bash
echo "Validating sample data"
shopt -s nullglob
for file in samples/*/TIDES/datapackage.json; do
    echo "Validating: $file"
    frictionless validate --schema-sync "$file"
done
shopt -u nullglob
```